README.md

```diff
@@ -1 +1,164 @@
-# amiapp-reactnative
\ No newline at end of file
+# Ami app
+
+This is a sample app using [Customer.io's React Native package](https://www.npmjs.com/package/customerio-reactnative). Ami app currently works only on iOS devices. The work to get it on Android is in progress.
+
+# What all features can I test with Ami app ?
+Ami app gives you flexibility to test multiple features such as :
+
+- Package initialisation
+- Identify user
+- Clear user identify
+- Track events
+- Device attributes (default & custom)
+- Profile attributes (default & custom)
+- Push Notifications 
+- Rich Push & deep links
+
+
+# Getting Started
+Ami app already includes all the dependencies required. To install the libraries using :
+```
+npm install
+```
+**iOS only**
+
+iOS needs an additional step to install libraries using CocoaPods
+```
+cd ios
+pod install
+cd ..
+```
+**Note** : Make sure that the deployment target is set to minimum 10.0 in Xcode.
+
+# Running the app
+**To run the app on iOS Simulator**
+
+Use the following commands to run the app on iOS Simulator
+
+```
+npx react-native run-ios
+```
+**To run the app on device**
+
+To run using terminal
+
+```
+react-native run-ios --device "Max's iPhone"
```

Comment:
Maybe consider using a placeholder instead of `Max's iPhone`, or indicate that `Max's iPhone` is an example and should be changed.

Reply:
Updated to `react-native run-ios --device "Your iPhone"` which makes it clear to select your iPhone device from the list of devices available.